projects/schematics/src/migrations/5_0/methods-and-properties-deprecations/data/generated-methods-and-properties.migration.ts

```diff
@@ -4778,6 +4918,20 @@ export const GENERATED_METHODS_AND_PROPERTIES_MIGRATION: MethodPropertyDeprecati
       comment:
         "// TODO:Spartacus - The 'setTheme' method's signature changed to: 'setTheme(  theme: string | undefined): void'",
     },
+    {
+      class: 'TrackingEventsComponent',
```

Comment:
duplicate (check line 2003)

Reply:
The move is properly registered in the doc and `rename-symbol.ts`